tests/test.py

```diff
@@ -56,6 +56,10 @@
 # storage = client.model(data)
```

Comment:
@hassaanfarooq01 Can you clean up these tests and make them permanent on another branch please.